src/mame/machine/nl_gtrak10.cpp

```diff
@@ -0,0 +1,380 @@
+	RES(R13, RES_K(6.8)) NET_C(R13.1, GROUND) NET_C(R13.2, J5.D4)
+	RES(R14, RES_K(6.8)) NET_C(R14.1, GROUND) NET_C(R14.2, J5.D3)
+	RES(R12, RES_K(6.8)) NET_C(R12.1, GROUND) NET_C(R12.2, J5.D2)
+	RES(R10, RES_K(6.8)) NET_C(R10.1, GROUND) NET_C(R10.2, J5.D1)
+	RES( R8, RES_K(6.8)) NET_C( R8.1, GROUND) NET_C( R8.2, J5.D0)
+	#endif
+
+	// ----------- Vertical positioning of the car -----------------
+	//     name,     EQ, MRQ,    S0Q,    S1Q,    S2Q,    S3Q,    D0,    D1,    D2,    D3
+	TTL_9314(H7, VLd1_Q,   P, GROUND, GROUND, GROUND, GROUND, DATA4, DATA5, DATA6, DATA7)
+	//     name,    CLK,   ENP,   ENT,     CLRQ,  LOADQ,      A,      B,      C,      D
+	TTL_9316(J7, HCOUNT,     P,     P, RESET1_Q, L5_A.Q,  H7.Q0,  H7.Q1,  H7.Q2,  H7.Q3)
+	TTL_9316(K7, HCOUNT, J7.RC,     P, RESET1_Q, L5_A.Q,      P,      P,      P,      P)
+	TTL_9316(L7, HCOUNT, J7.RC, K7.RC, RESET1_Q, L5_A.Q,      P, GROUND,      P,      P)
+	TTL_7400_NAND(L5_A, J7.RC, L7.RC)
+	ALIAS(1PHI2, J7.QB)
+	ALIAS(1PHI3, J7.QC)
+	ALIAS(1PHI4, J7.QD)
+
+	// ------------ Horizontal positioning of the car ----------------------
+	//     name,   CLK,   ENP,   ENT,      CLRQ, LOADQ,        A,      B,      C,      D
+	TTL_9316(E7, CLOCK,     P,     P,  RESET1_Q,  D5_B.Q, F7_A.Q, F7_D.Q, F7_C.Q, D3_C.Q)
+	TTL_9316(D7, CLOCK, E7.RC,     P,  RESET1_Q,  D5_B.Q,      P,      P,      P, GROUND)
+	TTL_7400_NAND(D5_B, E7.RC, B6_D.Q)
+	TTL_7404_INVERT(B6_D, H5_A.Q)
+	TTL_7410_NAND(H5_A, D7.RC, D6_A.Q, D6_B.Q) //weird: should be D6_B.QQ according to schematics
+	//        name,    CLK, J, K,     CLRQ
+	TTL_74107(D6_A,  D7.RC, P, P, RESET1_Q)
+	TTL_74107(D6_B, D6_A.Q, P, P, RESET1_Q)
+
+	TTL_7400_NAND(D3_C, VLd1, DATA3)
+	TTL_7408_AND(F7_C,  VLd1, DATA2)
+	TTL_7408_AND(F7_D,  VLd1, DATA1)
+	TTL_7408_AND(F7_A,  VLd1, DATA0)
+
+
+	// Car window:
+	// this flip-flop is incorrectly labeled "79" in the schematics
+	//       name,   CLK,      D, CLRQ,  PREQ
+	TTL_7474(A5_B, CLOCK, H5_A.Q,    P, L7.RC)
+	ALIAS(CAR_WINDOW,   A5_B.Q)
+	ALIAS(CAR_WINDOW_Q, A5_B.QQ)
+
+	// ------------- Car1Video shift registers: -----------
+	//      name,   CLK, CLK_INHIBIT_Q,   SH_LDQ,    SER,      A,     B,     C,     D,     E,     F,     G,     H
+	TTL_74165(H6, CLOCK,    CAR_WINDOW,  HSYNC_Q,      P,  DATA0, DATA1, DATA2, DATA3, DATA4, DATA5, DATA6, DATA7)
+	TTL_74165(F6, CLOCK,    CAR_WINDOW,     Ld1B,  H6.QH,  DATA0, DATA1, DATA2, DATA3, DATA4, DATA5, DATA6, DATA7)
+	TTL_7408_AND(F7_B, F6.QHQ, CAR_WINDOW_Q)
+	#ifdef CAR_WINDOW_HACK
+	ALIAS(CAR1VIDEO, CAR_WINDOW_Q)
+	#else
+	ALIAS(CAR1VIDEO, F7_B.Q)
+	#endif
+	TTL_7404_INVERT(L6_A, CAR1VIDEO)
+	ALIAS(CAR1VIDEO_Q, L6_A.Q)
+
+
+	// --------------- Hack ----------------------
+	// This is a signal only used for debugging.
+	// It generates a pattern of squares on the screen.
+	// CAR1VIDEO = not(16H & 16V | VBLANK | HBLANK)
+
+	#ifdef SIGNAL_RT_HACK
+	#ifdef SIGNAL_RT_HACK_XOR
+	TTL_7486_XOR(16H_xor_16V, 16H, 16V)
+	TTL_7402_NOR(nor, 16H_xor_16V, BLANK)
+	#else
```

Comment:
Please outdent preprocessor directives to the margin - they aren't affected by scope blocks.